docs/commands.rst

```diff
@@ -115,14 +115,14 @@ Create settings
 
 Command: **create-settings**
 
-Creates a new settings file based. If you call it without further arguments it
+Creates a new settings file-based. If you call it without further arguments it
 will create the settings in the default locations::
 
     wger create-settings
 
 If you pass custom paths, it's recommended to use absolute paths::
 
-    wger create-settings --settings-path /path/to/development/wger/settings-test.py --database-path /path/to/development/wger/database-test.sqlite
+    wger create-settings --settings-path /path/to/development/wger/settings-test.py --database-path /path/to/development/wger/database-test.SQLite
```

Comment:
this is part of the file name, better to leave in in lower case letters

Reply:
I just corrected my mistake.